### PR TITLE
Import V2 Map and Graph

### PR DESCRIPTION
**Pull request by bfinzer**
[#187134147] Bug fix: V2 map components and graph point properties are not being imported

**Note**: `v2-map-importer` and `v2-graph-importer` both contain commented out properties and will be fully integrated when certain decisions about color transparency, data-url image for background, and some others are made.

* Add v2MapImporter
* Register the V2TileImporter for maps
* Add ICodapV2MapStorage in codap-v2-types.ts
* Realized that not all graph properties were being imported, especially point display properties
* Added properties to `ICodapV2GraphStorage` for new legend properties and to store a plot background image
* Fine tuning `ICodapV2MapLayerStorage`
* Added plotBackgroundImage and plotBackgroundImageLockInfo to `GraphContentModel` even though we don't yet make use of them
* v2GraphImporter now imports point and background properties
* v2MapImporter now imports both point and polygon layers
* Some fine tuning to satisfy jest tests

### CODE REVIEW

File: v3/src/components/graph/v2-graph-importer.ts [hunk 1]
```diff
@@ -13,11 +12,25 @@ import {IAttributeDescriptionSnapshot} from "../data-display/models/data-configu
...
   if (!isV2GraphComponent(v2Component)) return
 
-  const {title = "", _links_: links, plotModels} = v2Component.componentStorage
+  const {
+    title = "", _links_: links, plotModels,
+
+    pointColor, strokeColor, pointSizeMultiplier,
+    strokeSameAsFill, isTransparent,
+    plotBackgroundImageLockInfo,
+/* The following are present in the componentStorage but not used in the V3 content model:
```
Comment (kswenson) [inline v3/src/components/graph/v2-graph-importer.ts:26]:
```suggestion
/* The following are present in the componentStorage but not used in the V3 content model (yet):
```

File: v3/src/v2/codap-v2-types.ts [hunk 2]
```diff
@@ -280,16 +282,19 @@ export interface ICodapV2GraphStorage extends ICodapV2BaseComponentStorage {
...
+  legendQuantilesAreLocked: boolean
   pointColor: string
   strokeColor: string
   pointSizeMultiplier: 1
   transparency: number
   strokeTransparency: number
   strokeSameAsFill: boolean
   isTransparent: boolean
-  plotBackgroundColor: string | null
+  plotBackgroundColor: string | undefined | null
```
Comment (kswenson) [inline v3/src/v2/codap-v2-types.ts:294]:
```suggestion
  plotBackgroundColor?: string | null
```

File: v3/cypress/fixtures/formula.codap3
[the remark this answers is not in the document]
Comment (bfinzer) [reply v3/cypress/fixtures/formula.codap3:1]: Actually this was deliberate, changing `plotBackgroundImageID` to `plotBackgroundImage`

**Review by kswenson — APPROVED** [review]
👍 Changes look good

**Pull request merged**